index.html

```diff
@@ -72,194 +72,146 @@ <h2 class="connect" href="#">LET'S CONNECT</h2>
       </div>
     </section>
 
-    <section class="work-container" id="containers">
-      <div class="container1" id="container1" data-modal-target="#modal1">
-        <img
-          class="desktop-image"
-          src="Images/Desktop-img1.png"
-          alt="Project Image"
-        />
-        <img
-          class="snapshoot"
-          src="Images/Snapshoot1.png"
-          alt="Project Image"
-        />
-        <div class="daily">
-          <h3 class="Tonic">Tonic</h3>
-          <ul class="offer">
-            <li class="canopy">canopy</li>
-            <span class="point"></span>
-            <li class="backend">Back End Dev</li>
-            <span class="point"></span>
-            <li class="year">2015</li>
-          </ul>
-          <p class="description">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-
-          <ul class="skills">
-            <li class="htl"><a href="#"></a>html</li>
-            <li class="css"><a href="#"></a>css</li>
-            <li class="js"><a href="#"></a>javascript</li>
-          </ul>
-          <button class="seeproject" data-modal-target="#modal1">See Project</button>
-        </div>
-      </div>
-
-      <div class="container2 desktop-reverse" id="container2" data-modal-target="#modal0">
-        <img
-          class="desktop-image"
-          src="Images/Desktop-img2.png"
-          alt="Project Image"
-        />
-        <img class="snapshoot" src="Images/Snapshoot2.png" alt="mobile image" />
-        <div class="daily">
-          <h2 class="Tonic2">Multi-Post Stories</h2>
-          <ul class="offer">
+    <section class="work-container" id="container">
+      <div class="container1" id="portfolio">    
+      <img class="desktop-image" src="Images/Desktop-img1.png" alt="Desktop Image">
+      <img class="snapshoot" src="Images/Snapshoot1.png" alt="mobile image">
+      <div class="daily">
+        <h3 class="Tonic">Tonic</h3>
+        <ul class="offer">
+           <li class="canopy">canopy</li>
+           <span class="point"></span>
+           <li class="backend">Back End Dev</li>
+           <span class="point"></span>
+           <li class="year">2015</li>
+        </ul>
+         <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+         
+         <ul class="skills">
+         <li class="htl"><a href="#"></a>html</li>
+         <li class="css"><a href="#"></a>css</li>
+         <li class="js"><a href="#"></a>javascript</li>
+         </ul>
+         <button class="seeproject">See Project</button>
+       </div>
+     </div>
+   
+     <div class="container2  desktop-reverse">  
```

Comment:
- [ ] According to the project requirement,
> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

- Your close button is not properly aligned as it is on the Figma design, please look into this as you need to replicate the Figma design as much as you can in this milestone. :wave: :airplane: 
- YOUR DESIGN:- 
![Selection_009](https://user-images.githubusercontent.com/107632780/232639030-9abef312-5855-4bd1-a148-aee7520ddc9a.png)

- FIGMA DESIGN:-
![Screenshot from 2023-04-18 01-14-58](https://user-images.githubusercontent.com/107632780/232639093-b43f04c2-7a3d-47c7-b282-9d3304b1177b.png)